cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/CubeSinkConfig.java

```diff
@@ -39,8 +39,8 @@
 
   @Name(Properties.Cube.DATASET_OTHER)
   @Description("Provide any dataset properties to be used if new Cube " +
-    "dataset needs to be created as a JSON Map. For example if aggregations are desired on fields - abc and xyz, the " +
-    "property should have the value: " +
+    "dataset needs to be created as a JSON Map. For example, if aggregations are desired on fields - abc and xyz," +
+    " the property should have the value: " +
```

Comment:
minor nit: put the space at the end of the previous line, and bring down an entire word or phrase if necessary tofit. If we always put the spaces at the end, it will make editing easier.